### ISSUE TITLE
Form Select Component does not work when used on relationship with searchable() enabled

### ISSUE THREAD

**Issue by kazak71** [laravel-filament/filament#14541]
### Package

filament/forms

### Package Version

v3.2.117

### Laravel Version

v11.28.0

### Livewire Version

v3.5.6

### PHP Version

PHP 8.2.22

### Problem description

I have an model Order with relationship customer (belongsTo Customer model)
In OrderResource I use this code:
```
Forms\Components\Select::make('customer_id')
    ->label('Customer')
    ->required()
    ->relationship('customer','name')
    ->searchable(['name'])
    ->createOptionForm(fn(Form $form)=>CustomerResource::form($form)),
```
When ->searchable(...) is used, the Select component does not work, but it works properly when ->searchable(...) is removed from options.
When Select component is used without relationship, searchable option works fine.

In browser dev tools there is an error;
```
Uncaught (in promise) TypeError: Cannot read properties of null (reading 'concat')
    at Proxy.getChoices (select.js?v=3.2.117.0:1:95896)
    at async Proxy.refreshChoices (select.js?v=3.2.117.0:1:95538)
    at async Proxy.init (select.js?v=3.2.117.0:1:94100)
```

### Expected behavior

This was working 3 days ago (before composer update) and It's critical bug because of impossibility to select from thousands customers or products and brakes app.

### Steps to reproduce

Forms\Components\Select::make('customer_id')
->label('Customer')
->required()
->relationship('customer','name')
->searchable(['name'])
->createOptionForm(fn(Form $form)=>CustomerResource::form($form)),

### Reproduction repository (issue will be closed if this is not valid)

https://github.com/kazak71/filament-select-test.git

**Comment by saulens22** [comment]
Same issue as in #14528

**Comment by mythsam** [comment]
As a walkaround for now, we can use ->preload() together with ->searchable() to make it work.

**Comment by kazak71** [comment]
> As a walkaround for now, we can use ->preload() together with ->searchable() to make it work.

I dont know how many records will be preloaded :)
And I have to update code in 98 places with preorder().
It was easier for me to downgrade (with backup) and stop updating.
But thanks for advice!

**Comment by wagnerfnds** [comment]
> > As a walkaround for now, we can use ->preload() together with ->searchable() to make it work.
> 
> I dont know how many records will be preloaded :) And I have to update code in 98 places with preorder(). It was easier for me to downgrade (with backup) and stop updating. But thanks for advice!

the main problem is that i already tried to downgrade the filament and livewire version to a 3 week ago version, and the erro still persist.

**Comment by ArcoVoets** [comment]
@wagnerfnds In #14528 a simple solution is mentioned, namely to downgrade Laravel to 11.27 or below.

**Comment by mythsam** [comment]
This issue should be fixed by v11.28.1.  Thanks to Laravel for the quick fix.

**Comment by saulens22** [comment]
Confirming as fixed in Laravel 11.28.1
